target/linux/rockchip/patches-5.4/005-rockchip-rk3328-add-compatible-to-NanoPi-R2S-etherne.patch

```diff
@@ -18,7 +18,7 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
  		#size-cells = <0>;
  
  		rtl8211e: ethernet-phy@1 {
-+			compatible = "ethernet-phy-id0143.bca2",
++			compatible = "ethernet-phy-id001c.c915",
```

Comment:
https://github.com/openwrt/openwrt/commit/9f907c46bb112788b9997dacd39bce261a8a1905

Reply:
The latest main branch already has this commit. Maybe you need to check it again.

Reply:
Thanks for the update.  It seems that David already picked up my comment on his commit.
I removed the PHY ID change, so we are left with the clock name, which actually the most important one.

Reply:
agree.  gmac_clkin +1